<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>341</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>RGB</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="rgb_line"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>CMYK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="cmyk_line"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>HSL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="hsl_line"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="message_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSelect_color">
    <property name="title">
     <string>Select color</string>
    </property>
    <addaction name="actionSelect_color"/>
   </widget>
   <addaction name="menuSelect_color"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSelect_color">
   <property name="text">
    <string>Color picker</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
